app/forms/sufia/forms/work_form.rb

```diff
@@ -4,6 +4,7 @@ class WorkForm < CurationConcerns::Forms::WorkForm
     include HydraEditor::Form::Permissions
 
     attr_reader :agreement_accepted
+    attr_accessor :file_sets
```

Comment:
Do you mean, "why attr_accessor instead of attr_reader"? It's because of the next pull request that I'll make where I allow the user to change the file title.